server/README.md

```diff
@@ -89,6 +89,8 @@ The INS server provides a backend RESTful service for INS-node and front-end web
     * `cmake -G "Unix Makefiles" -DCMAKE_BUILD_TYPE=Release ..`
     * `make`
     * `sudo make install`
+* Install spdlog
```

Comment:
I guess no need for this, since we removed the dependency to spdlog.

Reply:
Let's not change that in this pull request

Reply:
ok. but still needed to build.

Reply:
@lexious89 you shouldn't need to do that to build the ins_service component. The component builds successfully on CI node without the install spdlog step.

Reply:
Really? I added it because, it was what enabled a successful build for me (for a fresh installation). I was thinking about how fresh contributors can also build.

Reply:
Strange, it should not be necessary. I will also try it later at home. CI should be running in a [clean environment](https://github.com/platisd/indoor-navigation-system/blob/dev/.travis.yml) and running the [get_dependencies.sh](https://github.com/platisd/indoor-navigation-system/blob/dev/get_dependencies.sh) script.

Reply:
@lexious89 was right. Travis was building the tests only. A fix is introduced in #58